libraries/AP_Landing/AP_Landing_Deepstall.cpp

```diff
@@ -488,7 +511,7 @@ float AP_Landing_Deepstall::update_steering()
     ab.normalize();
     Vector2f a_air = location_diff(arc_exit, current_loc);
 
-    float crosstrack_error = a_air % ab;
+    crosstrack_error = a_air % ab;
     float sine_nu1 = constrain_float(crosstrack_error / MAX(L1_period, 0.1f), -0.7071f, 0.7107f);
```

Comment:
Sorry I should be clear. Deepstall definitely needs a different L1 period, so I would have needed an entire extra instance of AP_L1_Control, not just the parameter.